dev/src/v1/firestore_admin_client.ts

```diff
@@ -1156,29 +1231,83 @@ export class FirestoreAdminClient {
     });
     const callSettings = new gax.CallSettings(options);
     this.initialize();
-    return this._descriptors.page.listIndexes.createStream(
-      this._innerApiCalls.listIndexes as gax.GaxCall,
+    return this.descriptors.page.listIndexes.createStream(
+      this.innerApiCalls.listIndexes as gax.GaxCall,
       request,
       callSettings
     );
   }
+
+  /**
+   * Equivalent to {@link listIndexes}, but returns an iterable object.
+   *
+   * for-await-of syntax is used with the iterable to recursively get response element on-demand.
+   *
+   * @param {Object} request
+   *   The request object that will be sent.
+   * @param {string} request.parent
+   *   Required. A parent name of the form
+   *   `projects/{project_id}/databases/{database_id}/collectionGroups/{collection_id}`
+   * @param {string} request.filter
+   *   The filter to apply to list results.
+   * @param {number} request.pageSize
+   *   The number of results to return.
+   * @param {string} request.pageToken
+   *   A page token, returned from a previous call to
+   *   {@link google.firestore.admin.v1.FirestoreAdmin.ListIndexes|FirestoreAdmin.ListIndexes}, that may be used to get the next
+   *   page of results.
+   * @param {object} [options]
+   *   Call options. See {@link https://googleapis.dev/nodejs/google-gax/latest/interfaces/CallOptions.html|CallOptions} for more details.
+   * @returns {Object}
+   *   An iterable Object that conforms to @link https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Iteration_protocols.
+   */
+  listIndexesAsync(
```

Comment:
FYI @schmidt-sebastian: we now support this :)  `for await (const index of client.listIndexesAsync(request))` - feel free to use this in your manual layer!  (the same for any list* paginated method)

Reply:
Oh, awesome. Thanks for pointing this out. We should probably use expose this in our APIs that currently hide all paging, since we didn't want to use Streams or Callbacks as the primary way of interacting with our APIs.